packages/unpublished-relationship-performance-test-app/app/routes/materialization.js

```diff
@@ -1,7 +1,12 @@
 import Route from '@ember/routing/route';
 
+import CARS_PAYLOAD from '../utils/generate-fixtures-for-materialization-scenario';
+
 export default Route.extend({
   model() {
+    // having imported this ensures fixture generation
+    // is not measured within start-find-all to end-find-all
+    CARS_PAYLOAD;
```

Comment:
My initial feeling is the CARS_PAYLOAD is created when the import is evaluated, thus this might not be needed.  At least in es module land, it has to create a Module Record and execute it.  And, for example, if you import the same module twice, you don't execute it twice.  I think it does the same thing in cjs land as well.